### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by edubew**
# Capstone1 Project (Desktop Version)

## Project description
[Project Description](https://www.loom.com/share/a41eea4532d84520863f17482c7399b1)

### In this capstone project, the following was done:
- Add media queries to make the page responsive for all screen sizes
- Link the about page and logo to the home page 
- Added functionality on the hamburger menu such that it displays the nav items when clicked
- Added the featured speaker section dynamically with javascript
- Recorded a 5-minute loom video describing the main features of the project
- Deployed the page with GitHub pages
- Check to see that everything is working as planned

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -12,14 +12,24 @@
...
+
     <header>
       <nav>
         <div class="hambuger-menu">
           <span class="bars"></span>
           <span class="bars"></span>
           <span class="bars"></span>
         </div>
+        <a href="index.html" class="logo hide"><img src="images/OYA-logo.jpg" alt=""></a>
         <ul class="navbar-links hide">
```
Comment (rloterh) [inline index.html:33]: - [x] The red thick border around the _campaign_ text on the _white navbar_ is missing. Kindly ensure to add the same and it should be vertically aligned right with the _Logout_ menu item on the dark navbar. Remember we as much as possible want our design to mimic the given [template](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)

**Your design:**
image

**The original design:**
image

File: index.html [hunk 2]
```diff
@@ -31,7 +41,7 @@
     </header>
     <section class="introductory-section">
```
Comment (rloterh) [inline index.html:42]: - [x] The typography and line texts of your _hero_ section do not match the provided [desgn template](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). We actually want our design to as much as possible closely mimic the design of the provided template. The Home page should contain a _top_ section with a column that includes a **small line of text**, **the page title**, a **block of text**, a **medium line of text in bold letters**, and a **smaller line of text**, as can be seen in the provided template design. Let's ensure that the **font family**, font sizes, **font weights**, **font colors**, and **letter-spacing** match that of the provided template design. Also, ensure that the content in this section is centered horizontally (the extreme top and bottom space should be equal).

**Your hero design:**
image


**The original hero design:**
image

File: index.html [hunk 3]
```diff
@@ -41,22 +51,22 @@ <h2>Save the Date</h2>
         <h3>2022/09/29(Thurs) - 30th(Fri)</h3>
       </div>
     </section>
-</div>
     <section class="events-section">
```
Comment (rloterh) [inline index.html:55]: The spacing of your main program section does not follow the [template design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). Kindly ensure to improve upon the following:
- [x]  Contents in this section should have appropriate padding all around them. (See the indication from the attached screenshot)
- [x] The _link_ below the content of this section, _see the whole program_ should be in **uppercase**.
**Your main program section:**
image


**The original main program section:**
image

File: index.html [hunk 4]
```diff
@@ -75,62 +85,18 @@ <h3>Awards</h3>
                  <h3>Network</h3>
                  <p>Get opportunities to network with renown fashion icons and models and get useful insights from their pieces</p>
              </div>
+            </div>
+            <a href="" class="see-program hide">See the whole program</a>
              <button type="submit">Join OYA Summit</button>
         </div>
     </section>
     <section class="featured-speaker-section">
```
Comment (rloterh) [inline index.html:94]: The spacing/ alignment of your Featured Speakers sections does not match the provided design. Kindly note that we actually want this design to as much as possible mimic the provided  [design template](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). Kindly ensure to improve upon the following:
- [x] Add background decoration around each image
- [x] Shapes of images should appear more of a square
- [x] Ensure there are appropriate paddings around the elements of each content, as can be seen in the design template 
- [x] Ensure that the contents are spaced appropriately, as can be seen in the design template:

**Your Featured speakers section:**
image


**The original Featured speakers section:**
image

File: index.html [hunk 5]
```diff
@@ -12,14 +12,24 @@
     <title>Document</title>
   </head>
   <body>
-<div class="banner">
+      <div class="social-links hide">
```
Comment (rloterh) [inline index.html:15]: - [x] The mobile menu is missing from your mobile design screen. Kindly ensure to add a hamburger menu on the left side of the navigation for the mobile screen, which should display the mobile menu when clicked

File: script.js [hunk 6]
```diff
@@ -11,3 +11,79 @@ document.querySelectorAll('.nav-items').forEach((e) => e.addEventListener('click
...
+    Name: 'Patricia G',
+    title: 'Fitness Instructor',
+    description: '“To keep the body in good health is a duty… otherwise we shall not be able to keep our mind strong and clear.” “When it comes to eating right and exercising, there is no I will start tomorrow.” “It never gets easier, you just get better.”',
+    content: 'hide',
+    profileName: 'showSpeaker',
+  },
+];
+
+const allSpeakers = speakers.map((speaker) => `<div class="speaker ${speaker.content}${speaker.profileName}">
+    <img src="images/${speaker.featuredImage}" alt="">
```
Comment (rloterh) [inline script.js]: - [x] OPTIONAL
It's a good practice to provide `alt` names for your images so as to improve the accessibility features of your site.  Kindly endeavor to add `alt` names to **all** your images.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Edube, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦 🚦  

## Highlights 👍  
- Descriptive Pull Request
- Correct gitflow implemented
- Nice to acknowledge [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly check the comments marked **[OPTIONAL]** under the review._

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mahbubul14 — APPROVED** [review]
# Project Approved :trophy: :tada: 

Hi @edubew , 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Decriptive readme file.
:heavy_check_mark: Clean code.

### Optional suggestions :arrow_down_small: 

- [x] The commit message should be written in the imperative tense. `Ex: "Fix bug" instead of "Fixed bug".` [For more details.](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**